docs/index.rst

```diff
@@ -35,7 +35,7 @@ The framework is the main entry-point for teams to:
 
 To achieve this, RosTeamWS defines:
 
-#. an architecture of overlayed workspaces for sharing standard ROS packages;
+#. an architecture of overlaid workspaces for sharing standard ROS packages;
```

Comment:
This change should not be here, but in pre-commit fix PR.

Reply:
Done